dune_client/api/table.py

```diff
@@ -0,0 +1,99 @@
+"""
+Table API endpoints enables users to
+create and insert data into Dune.
+"""
+
+from __future__ import annotations
+from typing import List, Dict, Any, IO
+
+from dune_client.api.base import BaseRouter
+from dune_client.models import DuneError
+
+
+class TableAPI(BaseRouter):
+    """
+    Implementation of Table endpoints - Plus subscription only
+    https://docs.dune.com/api-reference/tables/
```

Comment:
Note: There is no landing page for the Tables API. This url redirects to https://docs.dune.com/api-reference/tables/endpoint/create and there only appears to be one direct link in the docs page (inside a comment on uploadCSV)

<img width="873" alt="image" src="https://github.com/duneanalytics/dune-client/assets/11778116/6fd0e1a6-4020-4a6b-a24c-7ea88d194a1b">

Reply:
Hmm, you are correct. 
Opening a ticket internally